docker/launch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+trap ctrl_c INT
+
+function ctrl_c() {
+    docker-compose down
```

Comment:
in my experience, ctrl-c on the `docker-compose up` process stops the containers but doesn't remove them, so it'll reuse them on the next run. i think we should create fresh containers each time